src/lib/hostapd.rb

```diff
@@ -9,776 +9,181 @@
 
 module Wifi
 
+  def self.start(config_path, hostapd_path)
+    networks = network_config config_path
+    # TODO: Is this rubyish?
+    if not networks.any?
```

Comment:
Personally, I prefer the "full-method wrap" over guard clauses, i.e.

```
if networks.any?
  all the code
end
```

This implies that you return nil instead of false, but since that's also falsy in a lot of situations it's better - but opinions on this differ, see [this recent discussion](https://github.com/colszowka/simplecov/issues/452) I had with a bunch of my fellow contributors for simplecov